lms/services/public_id.py

```diff
@@ -0,0 +1,50 @@
+from typing import Optional, TypeVar
+
+from lms.models import Region
+from lms.validation import ValidationError
+
+Model = TypeVar("Model")
+
+
+def get_by_public_id(  # pylint:disable=too-many-arguments
+    db, model: Model, public_id: str, region: Region, app="lms", type_="org"
```

Comment:
```suggestion
    db, model: Model, public_id: str, region: Region,  type_, app="lms"
```
If this is intended to be generic, it probably shouldn't specify type